package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Discord OIDC Wrapper in CF Workers",
   "main": "worker.js",
   "scripts": {
-    "start": "wrangler dev"
+    "start": "wrangler dev",
+    "publish": "wrangler publish",
+    "add_token": "wrangler secret put DISCORD_TOKEN"
```

Comment:
I appreciate these, but I think it falls outside of the scope of the PR and is something I would like to think about separately - I sorta like the idea of getting the user familiar with wrangler commands rather than hiding them from than, since it could make it more intuitive to understand how to debug if something goes wrong.

Reply:
yeah, i see what you mean